### PR TITLE
Bt security api renaming

### PR DESCRIPTION
**Pull request by joerchan**
Update security API naming.

### CODE REVIEW

File: include/bluetooth/conn.h [hunk 1]
```diff
@@ -279,15 +279,22 @@ struct bt_conn *bt_conn_create_slave_le(const bt_addr_le_t *peer,
 /** Security level. */
 typedef enum __packed {
 	/** Only for BR/EDR special cases, like SDP */
-	BT_SECURITY_NONE,
+	BT_SECURITY_L0,
 	/** No encryption and no authentication. */
```
Comment (carlescufi) [inline include/bluetooth/conn.h]: Can you add: `Level 1: No encryption and no authentication`
same with the rest

**Review by carlescufi — CHANGES_REQUESTED** [review]
Nice, thank you! Tiny change requested

File: include/bluetooth/conn.h (hunk 1 above)
Comment (joerchan) [reply include/bluetooth/conn.h]: Done.

**Review by jhedberg — APPROVED** [review]

**Review by Vudentz — APPROVED** [review]
LGTM

**Review by carlescufi — APPROVED** [review]

**Pull request merged**